agent/src/cli/command-bench/EvaluationDocument.ts

```diff
@@ -154,8 +158,8 @@ export class EvaluationDocument {
                     out.push(' EXACT_MATCH')
                 }
 
-                if (item.resultExact) {
-                    out.push(' EXACT_MATCH')
+                if (item.contextItems) {
+                    pushMultilineText('CONTEXT_ITEMS', item.contextItemsJSON ?? '[]')
```

Comment:
This should make the generated output more readable. Having the JSON formatted in a single line makes it unreadable in the generated snapshots. If we're not using the snapshots for manual review then we should skip adding `CONTEXT_ITEMS` here (it's still to the leaderboard via the CSV file)
```suggestion
                    pushMultilineText('CONTEXT_ITEMS', JSON.stringify(JSON.parse(item.contextItemsJSON ?? '[]'), null, 2))
```

Reply:
Hm, with your suggestion we're performing a redundant stringify + parse step (`item.contextItemsJSON` is already stringified). Can we go without them?

Reply:
If `item.contextItems` is available just stringify that. Performance is not a problem here though

Reply:
parsing and serializing json is cheap (most of the time), and this isn't a performance sensitive path anyways